core/jobs_test.py

```diff
@@ -808,6 +808,8 @@ def test_raise_error_with_get_continuous_computation_class(self):
             (
                 jobs.BaseMapReduceJobManagerForContinuousComputations.  # pylint: disable=protected-access
                 _get_continuous_computation_class()
+                # Access of private attribute OK
+                # because testing class structure.
```

Comment:
Can you move this to above the line it's referring to? I think it's pretty common to assume that a comment refers to the line(s) below the comment.

Reply:
Also don't break strings that are less than 80 chars, we break them right before 80. However, a more important question that I have here: How did you come to that conclusion?